CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 ### Fixed
 - [#5195](https://github.com/apache/trafficcontrol/issues/5195) - Correctly show CDN ID in Changelog during Snap
+- Fixed Traffic Router logging unnecessary warnings for IPv6-only caches
```

Comment:
Should this have the `[#5372]` in the line?

Reply:
We don't normally link the changelog entry to the PR, but we do sometimes link to the associated issue. In this case, there is no associated issue since I didn't create one 😄

Reply:
oh duh.